src/mmo/socket.js

```diff
@@ -452,15 +422,17 @@ export const sendChat = ({ message }) => {
     sendData(rootMsg.serializeBinary())
 }
 
-const redirect_uri = encodeURIComponent(url.protocol == "https:" ? 'https://sm64js.com' : 'http://localhost:9300')
+const redirect_uri = encodeURIComponent(process.env.NODE_ENV === 'rust'
+    ? `${url.protocol}//${window.location.host}`
+    : url.protocol == "https:" ? 'https://sm64js.com' : 'http://localhost:9300')
 
-const discord_client_id = process.env.DISCORD_CLIENT_ID
+const discord_client_id = "807123464414429184"
```

Comment:
ClientIDs are no secrets and they will be visible to any user that downloads the website's assets, so there is no need to store them in an env variable. Only the server has secrets

Reply:
you will also have to override these with your clientIDs

Reply:
I realize clientIDs are no secret... I just thought it was a good idea to have as environment variable to make it easier to change if that was ever needed...

Or if someone wanted to setup their own server, they could use their own clientID

Reply:
override with your clientIDs? what do you mean?

Reply:
ok gotcha, I will add it back.
I meant that you replace my clientID with yours, but I will add back the env variable